### ISSUE TITLE
https://reactstrap.github.io/components/ is broken, I seem to view any components

### ISSUE THREAD

**Issue by sdc2152** [reactstrap/reactstrap#952]
### Issue description

- components: `name`
- reactstrap version `#x.x.x`
- import method `umd/csj/es` (if umd, are you using the "full" version (only for v5+)?)
- react version `#x.x.x`
- bootstrap version `#x.x.x`

### Steps to reproduce issue

1. ...
2. ...

### Error message in console

```
paste error message with stacktrack here
```

**Comment by sdc2152** [comment]
Im not sure if this is the correct place to bring this up.

**Comment by wainola** [comment]
Yes, this part of the site is broken. I hope they fixed as soon as posible.

**Comment by mackbrowne** [comment]
Agreed, super broken.  Plz fix

**Comment by mackbrowne** [comment]
<img width="1275" alt="screen shot 2018-04-13 at 11 52 20 am" src="https://user-images.githubusercontent.com/1291776/38744937-28258b42-3f11-11e8-8968-92f73d634b45.png">


When i click 'components'

**Comment by ridhwaans** [comment]
same

**Comment by alucard87pl** [comment]
same

**Comment by DevCam** [comment]
You can use https://web.archive.org/web/20170907010948/http://reactstrap.github.io:80/components/alerts/
while its being fixed.

**Comment by mackbrowne** [comment]
ETA on fix?  Alternative to using the docs?

**Comment by TheSharpieOne** [comment]
I cannot seem to reproduce locally (though I am seeing the issue on the site). I'm just going to revert the last change for now.

**Comment by Fakkio** [comment]
Just for info, I don't know if you noticed, but i saw some kind of react error in console when it was down.
